src/components/PineTypes/Enum.js

```diff
@@ -23,7 +24,7 @@ export const rules = {
   'is not': (target, value) => target !== value
 }
 
-export const validate = _.isBoolean
+export const validate = value => !!value
```

Comment:
These aren't the same! Is that intentional?

Reply:
I snuck in a little hotfix here 😉 The `isBoolean` implementation is completely wrong.